docs/kind-example-config.yaml

```diff
@@ -3,8 +3,29 @@ apiVersion: kind.sigs.k8s.io/v1alpha1
 kind: Config
 # number of nodes in the cluster (currently only 1 is supported)
 numNodes: 1
-# template for kubeadm config, "" -> the default template
-kubeadmConfigTemplate: ""
+# these patches are applied to the generated kubeadm config
+# using `kustomize build` where the kustomization.yaml looks something like:
+# resources:
+# - generated-config.yaml
+# patches:
+# - file-containing-contents-of-the-first-patch.yaml
+# - file-containing-contents-of-the-second-patch.yaml
+# NOTE: this is somewhat brittle as the kubeadm config is not yet stable and
+# kind will need to select config versions relative to the kubernetes version
+# but it allows making advanced configuration when kind does not expose
+# any other options
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1alpha2
+  kind: MasterConfiguration
+  networking:
+    # Don't think setting pod subnet is currently required
```

Comment:
Not quite sure what the purpose of this comment is? Is this copied from my own config overlay in cert-manager? 😅

Reply:
removed this